### DISCUSSION TITLE
Disabling Registry

### DISCUSSION THREAD

**Discussion by Tobi-De** [caprover/caprover#2004] [Q&A] [answered]
Hi, I inadvertently enabled the registry in the cluster tab and now want to disable it.
The docs:

> Disabling Registry:
>
> At any point in time, you have the option to:
>
>    Disable Registry
>   Delete Registry Auth Details

https://caprover.com/docs/app-scaling-and-cluster.html#disabling-registry
But I can't find any option on the UI to do so.

**Comment by githubsaturn** [comment, marked as answer]
In front of `Docker Registry for Pushing New Images` there is a Pen Icon. :pen:  

Click on it and select Disable Push

**Reply by Tobi-De** [reply]
Thanks so much for thus
